.travis.yml

```diff
@@ -33,22 +33,21 @@ install:
       unzip protoc-3.0.0-${TRAVIS_OS_NAME}-x86_64.zip
     )
   - export PATH=$(pwd)/protoc/bin:$PATH
-  - go get -d -v -t ./...
+  - go get -d ./...
```

Comment:
Yes, we agreed that `-t` downloads the test packages for dependencies, but we don't ever invoke these tests, making this unneeded extra work. 
`-v` clutters the output when looking for errors.